intern.json

```diff
@@ -13,31 +13,30 @@
 	"functionalSuites": [
 		"./_build/tests/functional/**/*.js"
 	],
-	"loader": {
-		"script": "dojo2",
-		"options": {
-			"packages": [
-				{ "name": "src", "location": "_build/src" },
-				{ "name": "tests", "location": "_build/tests" },
-				{ "name": "cldr-data", "location": "node_modules/cldr-data" },
-				{ "name": "cldrjs", "location": "node_modules/cldrjs" },
-				{ "name": "@dojo", "location": "node_modules/@dojo" },
-				{ "name": "globalize", "location": "node_modules/globalize", "main": "dist/globalize" },
-				{ "name": "sinon", "location": "node_modules/sinon/pkg", "main": "sinon" }
-			],
-			"map": {
-				"globalize": {
-					"cldr": "cldrjs/dist/cldr",
-					"cldr/event": "cldrjs/dist/cldr/event",
-					"cldr/supplemental": "cldrjs/dist/cldr/supplemental",
-					"cldr/unresolved": "cldrjs/dist/cldr/unresolved"
+	"coverage": [
+		"./_build/src/**/*.js"
+	],
+	"browser": {
+		"loader": {
+			"script": "./node_modules/grunt-dojo2/lib/intern/internLoader.js",
```

Comment:
intern doesn't let us specify loader options without specifying the loader script, so we need to reference the loader in grunt-dojo2 manually in order for the packages below to take affect.